test/test_async.py

```diff
@@ -144,7 +144,18 @@ def handle_all_unhandled_exceptions(type, value, traceback):
             tickers = await exchange.fetch_tickers([symbol])
             dump(green(exchange.id), 'fetched', green(len(list(tickers.keys()))), 'tickers')
     else:
-        dump(yellow(exchange.id), 'fetching all tickers at once not supported')
+            dump(green(exchange.id), 'fetching all tickers by simultaneous multiple concurrent requests')
+            # Some exchanges not all the symbols can fetch tickers for
+            symbols_to_load = [symbol for symbol in exchange.symbols if not '.d' in symbol]
+            if exchange.id == 'bitmex':
+                symbols_to_load = ['BTC/USD', 'B_BLOCKSZ17', 'DASHZ17', 'ETC7D', 'ETHZ17', 'LTCZ17', 'XBJZ17', 'XBTZ17', 'XMRZ17', 'XRPZ17', 'XTZZ17', 'ZECZ17']
```

Comment:
Unfortunately, this line for Bitmex will break the test upon 7day contract expiration. Basically, their tickers change continuously, so we have to make another workaround for it. But, this isn't a big problem, will fix it anyway.

Reply:
Interesting yes, it was just a quick fix, as errors were occurring for attempting to fetch all of the individual tickers for each symbol contained in bitmex.symbols, it seems some of them are not supported, however still listed ? Keen to see your fix :)